src/metabase/notification/send.clj

```diff
@@ -377,17 +398,25 @@
       (ensure-enough-workers!)
       (put-notification! queue notification))))
 
-(defonce ^{:private true
-           :doc "Do not use this queue directly, use the dispatcher instead."}
-  notification-queue (delay (create-notification-queue)))
+(defonce ^:private dedup-priority-dispatcher
+  (delay (create-notification-dispatcher (notification-thread-pool-size) (create-dedup-priority-queue))))
+
+(defonce ^:private simple-blocking-dispatcher
+  (delay (create-notification-dispatcher (notification-thread-pool-size) (create-blocking-queue))))
```

Comment:
Let's create a new setting for the pool size, even if the default is to delegate to the other one.

Reply:
It's counter intuitive that increasing the one config option by `N` increase actual worker threads by `2N`

Reply:
oh right, I had a separate setting for its already, forgot to use it here. https://github.com/metabase/metabase/pull/57769/commits/9e0ace7d6869dff3e045659293e22ec7cceb9bf4